src/features/components/templates/navigation/template/desktop-menu/styles.scss

```diff
@@ -48,8 +48,6 @@
 
     &_scrollable_menu {
         @media screen and (max-height: 975px) {
-            block-size: 100vh;
```

Comment:
If I remember correctly we need these values in order for the menu to be scrollable for devices with small height.